src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -16,12 +16,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
             modelBuilder
                 .HasAnnotation("ProductVersion", "5.0.0-rc.1.20416.1");
 
-            modelBuilder.Entity("IdentityServer4.EntityFramework.Entities.DeviceFlowCodes", b =>
+            modelBuilder.Entity("Duende.IdentityServer.EntityFramework.Entities.DeviceFlowCodes", b =>
                 {
                     b.Property<string>("UserCode")
                         .HasMaxLength(200)
                         .HasColumnType("TEXT");
-
+                        
```

Comment:
Did EF add the extra whitespace❔

Reply:
No I copied some migrations from Duende but those didn't appear to be what we wanted, so I reverted them and messed up the whitespace will fix